api/src/main/java/marquez/graphql/Daos.java

```diff
@@ -0,0 +1,135 @@
+package marquez.graphql;
+
+import java.util.List;
+import java.util.UUID;
+import marquez.db.JobVersionDao.IoType;
+import marquez.graphql.mapper.ObjectMapMapper;
+import marquez.graphql.mapper.RowMap;
+import org.jdbi.v3.sqlobject.config.RegisterRowMapper;
+import org.jdbi.v3.sqlobject.statement.SqlQuery;
+
+@RegisterRowMapper(ObjectMapMapper.class)
+public interface Daos {
```

Comment:
_Minor_: I'm wondering if `GraphqlDao` would be a better name to best express the underlying quries?